src/App.tsx

```diff
@@ -15,37 +15,83 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortType {
+  ALPHABET = 'alphabet',
+  LENGTH = 'length',
+  REVERSE = 'reverse',
+}
+
 export const App: React.FC = () => {
+  const [visibleGoods, setVisibleGoods] = useState<string[]>(goodsFromServer);
+  const [activeSort, setActiveSort] = useState<SortType | ''>('');
+  const [isReversed, setIsReversed] = useState(false);
+
+  const resetGoodsOfOrder = () => {
+    setVisibleGoods(goodsFromServer);
+    setActiveSort('');
```

Comment:
```suggestion
    setActiveSort('');
    setIsReversed(false);
```

Reply:
<img width="620" alt="image" src="https://github.com/user-attachments/assets/0cd721f6-5137-46db-b10e-25cf48a2fbcb">
still not fixed